lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_action.rb

```diff
@@ -44,19 +51,24 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :repository,
                                        env_name: 'GHHELPER_REPOSITORY',
-                                       description: 'The remote path of the GH repository on which we work',
+                                       description: 'The slug (`<org>/<repo>`) of the GitHub repository we want to create the release on',
                                        optional: false,
                                        type: String),
           FastlaneCore::ConfigItem.new(key: :version,
                                        env_name: 'GHHELPER_CREATE_RELEASE_VERSION',
                                        description: 'The version of the release',
                                        optional: false,
-                                       is_string: true),
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :target,
+                                       env_name: 'GHHELPER_TARGET_COMMITISH',
+                                       description: 'The branch name or commit sha1 the new tag should point to - if that tag does not exist yet when publishing the release. If omitted, will default to the current HEAD commit at the time of this call',
```

Comment:
Fixed in caf7fbd7749c607f56a61ff6ad0757617f41d203 – also just used the name `SHA` because who knows maybe one day they'll migrate from SHA1 to SHA256…